{
  "info": {
    "author": "Daniel Perrefort",
    "author_email": "",
    "bugtrack_url": null,
    "classifiers": [
      "Natural Language :: English",
      "Operating System :: OS Independent",
      "Programming Language :: Python",
      "Programming Language :: Python :: 3",
      "Programming Language :: Python :: 3.6",
      "Programming Language :: Python :: 3.7",
      "Programming Language :: Python :: 3.8",
      "Programming Language :: Python :: 3.9"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "",
    "keywords": "Django HTTP Error Pages",
    "license": "GPL v3",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-easy-error",
    "package_url": "https://pypi.org/project/django-easy-error/",
    "platform": null,
    "project_url": "https://pypi.org/project/django-easy-error/",
    "project_urls": null,
    "release_url": "https://pypi.org/project/django-easy-error/1.0.1/",
    "requires_dist": [
      "django"
    ],
    "requires_python": ">=3.6",
    "summary": "Easy template integration for django error pages",
    "version": "1.0.1",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 14062760,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "2618a345032544dcec71246c1edd8134ee24104eb2da0dfc19b121d6794d7d29",
        "md5": "87deca60988fb3b5bc8327dca674e79b",
        "sha256": "16cfbd0dd212eec53017e471e362313e1a272621a4ec04b880f4654408656dd3"
      },
      "downloads": -1,
      "filename": "django_easy_error-1.0.1-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "87deca60988fb3b5bc8327dca674e79b",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.6",
      "size": 17006,
      "upload_time": "2022-06-01T20:54:23",
      "upload_time_iso_8601": "2022-06-01T20:54:23.272916Z",
      "url": "https://files.pythonhosted.org/packages/26/18/a345032544dcec71246c1edd8134ee24104eb2da0dfc19b121d6794d7d29/django_easy_error-1.0.1-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "7381f9aca98d28e3f0d336b9e1bc9ab4a0a09f72e44abda068842d3f5c8f4ef9",
        "md5": "b40e92e9e17dc919f70c06e0f5c71571",
        "sha256": "c997c8db8495d7cd0581c93bc98bdba01d43940b864dcda3b206dfc596dd7656"
      },
      "downloads": -1,
      "filename": "django-easy-error-1.0.1.tar.gz",
      "has_sig": false,
      "md5_digest": "b40e92e9e17dc919f70c06e0f5c71571",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": ">=3.6",
      "size": 16427,
      "upload_time": "2022-06-01T20:54:24",
      "upload_time_iso_8601": "2022-06-01T20:54:24.395990Z",
      "url": "https://files.pythonhosted.org/packages/73/81/f9aca98d28e3f0d336b9e1bc9ab4a0a09f72e44abda068842d3f5c8f4ef9/django-easy-error-1.0.1.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}